config/engines.yml

```diff
@@ -25,6 +25,12 @@ duplication:
   community: false
   enable_regexps:
   default_ratings_paths:
+  default_config:
```

Comment:
That sounds like a good approach! I'll extract out some defaults in a separate PR.